src/SwedbankPay.Sdk/PaymentInstruments/Swish/SwishPaymentSaleRequest.cs

```diff
@@ -1,24 +1,26 @@
-﻿namespace SwedbankPay.Sdk.Payments.SwishPayments
-{
-    public class SwishPaymentSaleRequest
-    {
-        public SwishPaymentSaleRequest(Msisdn msisdn)
-        {
-            Transaction = new SwishPaymentSaleTransaction(msisdn);
-        }
-
-
-        public SwishPaymentSaleTransaction Transaction;
-
-        public class SwishPaymentSaleTransaction
-        {
-            protected internal SwishPaymentSaleTransaction(Msisdn msisdn)
-            {
-                Msisdn = msisdn;
-            }
-
-
-            public Msisdn Msisdn { get; }
-        }
-    }
-}
+﻿using SwedbankPay.Sdk.Common;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Swish
+{
+    public class SwishPaymentSaleRequest
+    {
+        public SwishPaymentSaleRequest(Msisdn msisdn)
+        {
+            Transaction = new SwishPaymentSaleTransaction(msisdn);
+        }
+
+
+        public SwishPaymentSaleTransaction Transaction;

```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)